File: labServiceUIJava/labServiceUIJava/src/main/java/com/example/labserviceuijava/service/ServiceCashier.java
package com.example.labserviceuijava.service;

import com.example.labserviceuijava.model.Cashier;
import com.example.labserviceuijava.repository.CashierDBRepository;
import com.example.labserviceuijava.repository.exceptions.RepositoryException;
import com.example.labserviceuijava.validator.CashierValidator;

public class ServiceCashier {

    private final CashierDBRepository cashierDBRepository;

    private final CashierValidator cashierValidator;

    public ServiceCashier(CashierDBRepository cashierDBRepository, CashierValidator cashierValidator) {
        this.cashierDBRepository = cashierDBRepository;
        this.cashierValidator = cashierValidator;
    }

    public boolean getCashierByUserPassword(String username, String password){
        Cashier cashier=cashierDBRepository.getCashierByUserPassword(username,password);
        return cashier!=null;
    }

}

File: labServiceUIcs/WindowsFormsApp2/service/ServiceCashier.cs
using WindowsFormsApp2.model;
using WindowsFormsApp2.repository;

namespace WindowsFormsApp2
{
    public class ServiceCashier
    {
        private CashiersDBRepository cashiersDbRepository;

        public ServiceCashier(CashiersDBRepository cashiersDbRepository)
        {
            this.cashiersDbRepository = cashiersDbRepository;
        }

        public int getCashierByUserPassword(string username, string password)
        {
            Cashier cashier = cashiersDbRepository.getCashierByUsernamePassword(username, password);
            if (cashier == null)
                return -1;
            return 1;
        }
    }
}